src/isPalindrome.js

```diff
@@ -20,6 +20,18 @@
  */
 function isPalindrome(str) {
   // write code here
+  if (str === '') {
+    return true;
+  }
+  const reg1 = /\w/g;
+  const smallStr = str.toLowerCase().match(reg1).join('');
+  const revStr = str.toLowerCase().match(reg1).reverse().join('');
+
+  if (smallStr === revStr) {
```

Comment:
you can write like this 
```suggestion
  return smallStr === revStr;
```